pkg/kubectl/stop.go

```diff
@@ -337,6 +333,8 @@ func (reaper *StatefulSetReaper) Stop(namespace, name string, timeout time.Durat
 	}
 	if timeout == 0 {
 		numReplicas := ss.Spec.Replicas
+
+		// BUG: this timeout is never used.
```

Comment:
@foxish would you mind looking at this? Your name come up as a recent editor of this function.